README.md

```diff
@@ -11,7 +11,7 @@ RVW Developers Core - SFCC Cartridge
 
 - [X] Execute Server Side JavaScript on your Sandbox in your favorite browser
 - [X] No Site Preferences or Misc Imports Required, just drop in and go
-- [X] Safety Measures to prevent running in Production Environments
+- [X] Safety Measures to prevent running in Production and Staging Environments
```

Comment:
It can run in staging

Reply:
I added this check instead `if (System.getInstanceType() !== System.DEVELOPMENT_SYSTEM) {`
This will not allow it to run on PROD nor on STG (since it's just as dangerous due to data replication happening usually on a schedule to PROD).
So it only works on sandbox/development. 
I can revert that and allow it to run on STG as well, but it might not be desired.

Reply:
we want this to work in staging. I've actually been using this for about 3 years now, and use it in staging and development just about as often as in my sandbox.

Reply:
done